Modules/System/Cryptography Management/src/Xml/SignedXmlImpl.Codeunit.al

```diff
@@ -222,4 +240,4 @@ codeunit 1461 "SignedXml Impl."
         SignedXmlType := GetDotNetType(DotNetSignedXml);
         exit(CopyStr(SignedXmlType.GetField(FieldName).GetValue(GetDotNetType(DotNetSignedXml)), 1, 250));
     end;
-}
\ No newline at end of file
+}
```

Comment:
to keep things consistent, lets remove the new line at end of file

Reply:
fixed